rosbridge_library/src/rosbridge_library/protocol.py

```diff
@@ -118,6 +118,9 @@ def incoming(self, message_string=""):
         message_string -- the wire-level message sent by the client
 
         """
+        if type(message_string) is bytes:
```

Comment:
This decoding should be done at the rosbridge_server level before it makes it to this point, unless you are implementing support for incoming binary messages in this PR, which does not appear to be the case.

Reply:
I have moved it into `websocket_handler.py`